README.md

```diff
@@ -18,3 +18,16 @@ Some tricks and templates are available in [Templates](Templates/) folder.
   1. #### Refer to the [GSSoC Wiki Page](https://github.com/sahilbansal17/Competitive_Coding/wiki/GSSoC) for contribution guidelines.
 
   2. #### Refer to the [Coding Standards Wiki Page](https://github.com/sahilbansal17/Competitive_Coding/wiki/Coding-Standards) for the coding standards to be followed. 
+
+### How to contribute to Hacktoberfest ?
+  1. #### Register for Hacktoberfest. Visit https://hacktoberfest.digitalocean.com/sign_up/register
```

Comment:
```suggestion
  1. #### Register for Hacktoberfest. Visit [Hacktoberfest page](https://hacktoberfest.digitalocean.com/sign_up/register).
```

Reply:
Please use links in markdown as `[link name](link)` so that it gets hyperlinked.

Reply:
Please review latest commit d74295a